Debug for Django tests (#502)
tl-its-umich-edu/canvas-course-manager-next: README.md

Use `^C` to stop the container and `docker-compose down` to remove the last used image from staging.

### Debugging using VS Code
There are launch settings in this project for debugging both tests and the backend in VS Code. The ports 5678 and 5679 are opened by docker-compose for debugging. 

After you start up the application 1 port is opened by default in 5678 to debug the backend. You can go into the debugger panel and launch CCM Django to connect to this.

> To debug tests, it's easiest to just change the DEBUG_REMOTE_PORT on the command line to a different address and connect to that. ose is 5679. So for example you would run.

Reviewer: ```suggestion
To debug tests, it's easiest to just change the DEBUG_REMOTE_PORT on the command line to a different address and connect to that with port 5679. So for example you would run.
```
